5.machine_learning/0.train_logistic_regression/scripts/2.train_supp_models.py

```diff
@@ -0,0 +1,263 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ## Train machine learning models where we split the features by organelle
+# 
+# In this notebook, we will split the 625 features used in the overall final and shuffled models into two groups:
+# 
+# 1. Only F-actin features
+# 2. The rest of the features (e.g., nucleus, ER, Golgi/plasma membrane, and Mitochondria)
```

Comment:
You might want to specify how you handled the `correlation` features

Reply:
Good point, added!